android/expoview/build.gradle

```diff
@@ -99,6 +98,7 @@ repositories {
 
 android {
   compileSdkVersion safeExtGet("compileSdkVersion", 30)
+  ndkVersion "23.0.7421159"
```

Comment:
any reason we should upgrade ndk version here?

Reply:
yes, no need this line, i have checked it on my local